go.mod

```diff
@@ -1,16 +1,23 @@
 module github.com/digitalocean/clusterlint
 
-go 1.12
+go 1.14
```

Comment:
Bumping this means that we expect consumers of clusterlint to run on 1.14 (or later). Staying close to the latest version is usually fine with packages that are to be used as binaries only (e.g., CLIs). However, I believe that clusterlint can also be consumed as a library which means that we could potentially break users if we started to use 1.14+ only features. (See also [the docs on the `go` directive](https://golang.org/ref/mod#tmp_12).)

I wonder if we should stick to 1.12 for now. 🤔  We could also start to mimic Go's deprecation policy -- I _think_ they stop shipping security / bug fixes after a while, but I couldn't find the exact number of releases they support.

Reply:
+1 - let's stick with 1.12 for now, just in case anyone else is consuming clusterlint as a dependency.